src/Apps/Show/Components/ShowArtworks.tsx

```diff
@@ -45,7 +45,6 @@ const ShowArtworksFilter: React.FC<ShowArtworksFilterProps> = props => {
       filters={filters}
       sortOptions={[
         { text: "Gallery Curated", value: "partner_show_position" },
-        { text: "Default", value: "-decayed_merch" },
```

Comment:
deleted based on [this comment](https://artsyproduct.atlassian.net/browse/FX-4540?focusedCommentId=54198)